src/passes/GenerateDynCalls.cpp

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2020 WebAssembly Community Group participants
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//
+// Create thunks for use with emscripten Runtime.dynCall. Creates one for each
```

Comment:
The `Runtime` object was removed.
```suggestion
// Create thunks for use with emscripten's dynCall(). Creates one for each
```